src/table/styles.ts

```diff
@@ -102,9 +136,21 @@ export const StyledHeadRow = styled(TableRow, {
   },
 }));
 
+export const StyledHeadCell = styled(TableCell)({
+  ...cellCommon,
+  lineHeight: '150%',
+});
```

Comment:
I've added the similar style in my PR but never mind I'll update it with yours after you merge it.

Reply:
ok